code/modules/mob/living/simple_animal/hostile/terror_spiders/terror_spiders.dm

```diff
@@ -0,0 +1,328 @@
+
+var/global/list/ts_ckey_blacklist = list()
+var/global/ts_count_dead = 0
+var/global/ts_count_alive_awaymission = 0
+var/global/ts_count_alive_station = 0
+var/global/ts_death_last = 0
+var/global/ts_death_window = 9000 // 15 minutes
+var/global/list/ts_spiderlist = list()
+
+// --------------------------------------------------------------------------------
+// --------------------- TERROR SPIDERS: DEFAULTS ---------------------------------
+// --------------------------------------------------------------------------------
+// Because: http://tvtropes.org/pmwiki/pmwiki.php/Main/SpidersAreScary
+
+/mob/living/simple_animal/hostile/poison/terror_spider/
+	// Name / Description
+	name = "terror spider"
+	var/altnames = list()
+	var/name_usealtnames = 0 // if 1, spiders use their randomized names, if not they're all "<color> terror"
```

Comment:
The above system comes across as needlessly complex--it's better when a consistent item is called by a consistent name and not just because "oh, let's pick this name today for no reason".